src/components/dashboard/main-content/course-enrollments/course-cards/InProgressCourseCard.jsx

```diff
@@ -196,12 +198,9 @@ export const InProgressCourseCard = ({
       courseRunId: response.courseRunId,
       newStatus: response.courseRunStatus,
     });
+    addCourseEnrollmentStatusChangeAlert(COURSE_STATUSES.savedForLater);
     navigate('.', {
       replace: true,
-      state: {
-        markedSavedForLaterSuccess: true,
-        markedInProgressSuccess: false,
-      },
     });
```

Comment:
[sanity check] Is this `navigate` call needed at all if no longer updating the location `state`? It's largely a no-op at this point, I believe.